src/shared/components/modal/modal.css

```diff
@@ -0,0 +1,42 @@
+.hotstream-modal-container {
+  width: 250px;
+  min-height: 130px;
+  background-color: #1e1e1e;
+}
```

Comment:
Suggestion: Use relative units and color variable

Reply:
I will change it